.github/sync.yml

```diff
@@ -2,6 +2,7 @@ group:
   - files:
       - source: _extensions/seedcase-theme/
         dest: _extensions/seedcase-project/seedcase-theme/
+        deleteOrphaned: true
```

Comment:
Just for my own understanding: what does this do?

Reply:
This compares all files between the folders and if there are some that are left over (like the favicon files when we moved them), this option will delete those left over files.